wasm-rpc/src/lib.rs

```diff
@@ -53,9 +53,28 @@ pub use extractor::{WitNodePointer, WitValueExtractor};
 #[cfg(feature = "stub")]
 pub use bindings::golem::rpc::types::{NodeIndex, RpcError, Uri, WasmRpc, WitNode, WitValue};
 
+#[cfg(not(feature = "stub"))]
```

Comment:
So, these are not required. 
The user is expected to either
- use only the `stub` feature flag,
- or the `host` feature flag that enables all host features
- or a combination of the host features

So you only need to check for`typeinfo`.

That said, annotating these imports looks very hard to maintain and fragile and I would either just use the fully qualified names, or if it is annoying, I would create a new `type_annotated_value` module, annotate the whole module with the `typeinfo` flag, and then just import them to the root crate:

lib.rs:
```rust
#[cfg(feature = "typeinfo")]
mod type_annotated_value;  // note it is private

#[cfg(feature = "typeinfo")]
pub use type_annotated_value::*; // things imported to the root crate and published
```

type_annotated_value.rs:
```rust
// no need for any flag checks
```

Reply:
haha we said the same thing

Reply:
Makes sense!

Reply:
Done